OtripleS.Web.Api.Tests.Unit/Services/Foundations/TeacherContacts/TeacherContactServiceTests.Exceptions.RetrieveById.cs

```diff
@@ -60,23 +58,21 @@ public async Task ShouldThrowDependencyExceptionOnRetrieveWhenSqlExceptionOccurs
         public async Task ShouldThrowDependencyExceptionOnRetrieveWhenDbUpdateExceptionOccursAndLogItAsync()
         {
             // given
-            var randomContactId = Guid.NewGuid();
-            var randomTeacherId = Guid.NewGuid();
-            Guid inputContactId = randomContactId;
-            Guid inputTeacherId = randomTeacherId;
+            Guid someContactId = Guid.NewGuid();
+            Guid someTeacherId = Guid.NewGuid();
             var databaseUpdateException = new DbUpdateException();
 
             var expectedTeacherContactDependencyException =
                 new TeacherContactDependencyException(databaseUpdateException);
 
             this.storageBrokerMock.Setup(broker =>
-                broker.SelectTeacherContactByIdAsync(inputTeacherId, inputContactId))
+                broker.SelectTeacherContactByIdAsync(someTeacherId, someContactId))
                     .ThrowsAsync(databaseUpdateException);
 
             // when
             ValueTask<TeacherContact> retrieveTeacherContactTask =
                 this.teacherContactService.RetrieveTeacherContactByIdAsync
-                (inputTeacherId, inputContactId);
+                (someTeacherId, someContactId);
```

Comment:
Need a tab here.

Reply:
@ShriHumrudha Fixed formatting.

Reply:
Just this line needed tab. Please revert all the alignment changes done in the second commit.